app/models/division.rb

```diff
@@ -90,6 +90,7 @@ class Division < ActiveRecord::Base
   validates :parent, presence: true, if: -> { Division.root.present? && Division.root_id != id }
 
   scope :by_name, -> { order("LOWER(divisions.name)") }
+  scope :by_public, -> { where(public: true) }
```

Comment:
I think this should just be `public`

Reply:
`public` is a keyword in rails @meltheadorable